tur/stern/build.sh

```diff
@@ -0,0 +1,40 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/stern/stern
+TERMUX_PKG_DESCRIPTION="⎈ Multi pod and container log tailing for Kubernetes -- Friendly fork of https://github.com/wercker/stern"
+TERMUX_PKG_LICENSE="Apache License 2.0"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="@idj0"
+TERMUX_PKG_VERSION=1.28.0
+TERMUX_PKG_SRCURL=https://github.com/stern/stern/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=7d0914cc3a3b884cce5bcbeb439f5c651c72f0737ba9517b663d7f911804489e
+TERMUX_PKG_DEPENDS="pacman, kubectl"
+TERMUX_PKG_BUILD_DEPENDS="golang"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+
+termux_step_make() {
+	termux_setup_golang
+
+	mkdir bin
+	go build -o ./bin ./...
+
+	mkdir -p completions/zsh
+	./bin/stern --completion zsh >completions/zsh/_stern
```

Comment:
The target of `./bin/stern` is `$TERMUX_ARCH-linux`. It may be able to execute because of qemu translation layer, but it is not a proper way.

Please ref https://github.com/termux/termux-packages/blob/master/packages/starship/build.sh#L49 to see how to properly generate and install shell completions.

Reply:
ok got it i'll use the postinst instead, thankyou for pointing out